### PR TITLE
Add support for highlighting CSP, HSTS, and HPKP

### PR DESCRIPTION
**Pull request by papandreou**
Adds the missing pieces to https://github.com/PrismJS/prism/pull/1016. I couldn't push to the existing branch, so I'm opening a fresh PR instead.

### CODE REVIEW

File: tests/run-child.js [hunk 1]
```diff
@@ -15,7 +15,10 @@ if (argv.language) {
 				}
 				process.send({success: true});
 			} catch (e) {
-				process.send({error: JSON.stringify(e)});
+				process.send({error: JSON.stringify({
+					message: e.message,
+					stack: e.stack
+				})});
```
Comment (Golmote) [inline tests/run-child.js]: Please don't include this change here and rather make another PR for it.

**Comment by Golmote** [comment]
Hi @papandreou! Thanks for working on this.
From a quick look, this seems quite complete to me. I'll have to give it a try before merging though.

I think running `gulp` should make changes to the Show Language plugin. Could you include them in the PR?

@ScottHelme Let me know if you're ok with this being merged instead of #1016. Note that @papandreou referenced you as the "owner" of the languages. This means we expect you to do some kind of maintenance if future issues occur on these components. Please discuss it if @papandreou should have this responsability instead.

**Comment by papandreou** [comment]
@Golmote, thanks for taking a look! I moved the error propagation thing to a separate PR (https://github.com/PrismJS/prism/pull/1276) and force pushed my branch here.

Also pushed commits that build the minified components and the "Show Language" plugin.

**Comment by ScottHelme** [comment]
All good on my side!

**Comment by papandreou** [comment]
I can help out as well, but @ScottHelme is certainly riding the bleeding edge more than I am :)

**Comment by Golmote** [comment]
Thank you both for contributing!
